commatrix/generate.go

```diff
@@ -183,11 +193,27 @@ func GenerateMatrixDiff(mat1 types.ComMatrix, mat2 types.ComMatrix) string {
 		}
 
 		if !mat1.Contains(cd) {
+			// add "-" before cd's mat1 doesn't contain but mat2 does
 			diff += fmt.Sprintf("- %s\n", cd)
 		}
 	}
 
-	return diff
+	return diff, nil
+}
+
+func getComMatrixColTagsByFormat(format string) string {
```

Comment:
I'm not sure about the name of the func, it says "get cols" while cols are specific to csv and it can generate the headers of a json/yaml formats as well where these are not columns

Reply:
I agree, maybe `getComMatrixHeaderTagsByFormat` is a better name. what do you think?

Reply:
getComMatrixHeadersByFormat makes sense to me